docs/_resources/troubleshooting.md

```diff
@@ -318,6 +319,12 @@ Applicable versions: **0.1 - 0.1.1** (fixed in **0.2**)
 
 ---
 
+## Power BI: Reports are not showing data outside of RangeStart/RangeEnd
```

Comment:
I don't think people will think about the fact that RangeStart/End are set when they don't see data. Can you add this to the "Reports are empty" section? It would also be good to add it to the troubleshooting walkthrough.

Reply:
I moved this to the reports are empty section and created a new PR add this to the guide as well. I simply added a step 3 with the instructions under the Validate your Power Bi config section. - https://github.com/microsoft/finops-toolkit/pull/969